<ui version="4.0" >
 <class>QUMediaPlayer</class>
 <widget class="QWidget" name="QUMediaPlayer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>79</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item row="0" column="0" >
      <widget class="QToolButton" name="prevBtn" >
       <property name="text" >
        <string>prev</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QToolButton" name="nextBtn" >
       <property name="text" >
        <string>next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QComboBox" name="sourceCombo" >
       <item>
        <property name="text" >
         <string>(source)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QToolButton" name="playBtn" >
       <property name="text" >
        <string>play</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="stopBtn" >
       <property name="text" >
        <string>stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QCheckBox" name="shuffleChk" >
       <property name="text" >
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="artistLbl" >
       <property name="text" >
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleLbl" >
       <property name="font" >
        <font>
         <pointsize>13</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeLbl" >
       <property name="text" >
        <string>0:00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QUAutoCue" name="autocue" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QUAutoCue</class>
   <extends>QTextBrowser</extends>
   <header>QUAutoCue.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
